common-utils/src/main/java/com/stnetix/ariaddna/commonutils/exeption/AriaddnaExeption.java

```diff
@@ -0,0 +1,8 @@
+package com.stnetix.ariaddna.commonutils.exeption;
+
+/**
+ * This class is the parent of all exceptions in the Ariaddna project.
+ */
+public class AriaddnaExeption extends Exception {
```

Comment:
:) ...
We need make constructor:
- AriaddnaException(Throwable cause)
- AriaddnaException(String traceMessage)
- AriaddnaException(String traceMessage, Throwable cause)
- AriaddnaException(String traceMessage, String errorMessage)
- AriaddnaException(String traceMessage, String errorMessage, Throwable cause)

And method for get error message from last cause:
- public String getErrorMessage()